standards/core/relay-sharding.md

```diff
@@ -166,9 +175,19 @@ This example node is part of shards `13`, `14`, and `45` in the Status main-net
 
 ## Automatic Sharding
 
-Autosharding selects shards automatically and is the default behavior for shard choice.
-The other choices being static and named sharding as seen in previous sections.
-Shards (pubsub topics) SHOULD be computed from content topics with the procedure below.
+Autosharding is an extension of static sharding, 
+where shards can automatically be selected based on content topic.
+This is only possible in cases where there exists a clear definition of a "network" combining multiple shards
+with a predefined cluster ID and number of shards in the network (`number_of_shards_in_network`).
+An example is [the Waku Network](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/64/network.md).
+To use autosharding,
+the shards corresponding to the network definition MUST be numbered from `0` and monotonously increase to `number_of_shards_in_network - 1`.
+The Waku Network SHOULD be considered the default destination for all app-level protocols.
+In cases where the Waku Network is not used but autosharding is desired,
+the default number of shards in a network SHOULD be considered `1`.
```

Comment:
I'd like to understand this sentence. 
Why would someone need `autosharding` with number of shard to be `1`?

Reply:
Because it allows usage of `content topic` only APIs, where `shard id` is never exposed or used by the developer.